src/Login.css

```diff
@@ -0,0 +1,66 @@
+//We also need to confirm these, since wireframes and Figma haven't been confirmed yet -- maybe just add a TODO comment:
+//TODO: confirm once mockups provided
```

Comment:
instead of
//comments
this should be 
/* comment */

Errors: 
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/126940746/224444822-82f9fd57-cdf5-4172-a3f8-cdb376a735cc.png">